app/views/webhooks/message.njk

```diff
@@ -54,7 +54,7 @@
   }) }}
 
   <div class="govuk-body">
-    <details class="govuk-details" data-module="govuk-details">
+    <details class="govuk-details" >
```

Comment:
Tiny thing - there's a stray space in this element.

Reply:
Good spot - fixed.